libs/remix-ui/static-analyser/src/lib/Button/StaticAnalyserButton.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'  //eslint-disable-line
+
+interface StaticAnalyserButtonProps {
+  onClick: (event) => void
+  buttonText: string,
+  disabled?: boolean
+}
+
+const StaticAnalyserButton = ({
```

Comment:
Let's also make this button a reusable component outside static analyser component.